src/Management/src/OpenCensus.Abstractions/Steeltoe.Management.OpenCensus.Abstractions.csproj

```
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) != 'Windows_NT'">netstandard2.0</TargetFrameworks>
+    <Description>OpenCensus .NET API abstractions</Description>
+    <IncludeSymbols>True</IncludeSymbols>
+    <PackageTags>Tracing;OpenCensus;Management;Monitoring</PackageTags>
+    <PackageIconUrl>https://opencensus.io/images/opencensus-logo.png</PackageIconUrl>
+    <PackageProjectUrl>https://opencensus.io</PackageProjectUrl>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <Authors>OpenCensus authors</Authors>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <RootNamespace>OpenCensus</RootNamespace>
+  </PropertyGroup>
```

so the package gets versioned right
```suggestion
    <VersionPrefix>$(VersionPrefix)</VersionPrefix>
    <VersionSuffix>$(VersionSuffix)</VersionSuffix>
  </PropertyGroup>
```